docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -525,6 +525,123 @@ as follows:
 	meta-analysis for the effect size causing a slightly non-symmetrical 
 	confidence interval around the mean.
 
+**Birth Weight**
+
+The effect of maternal consumption of iron fortified food on infant birth 
+weight was obtained from Haider et al. (2013). According to this data source,  
+birth weight among babies born to mothers increased, on average, by 15.1 grams 
+(95% CI: 6.0, 24.2) per 10 mg of daily iron consumption. The distribution of 
+the parameter should be modeled as follows:
+
+.. code-block:: Python
+
+	from scipy.stats import norm
+
+	# mean and 0.975-quantile of normal distribution for mean difference (MD)
+	mean = 15.1
+	q_975 = 24.2
+
+	# 0.975-quantile of standard normal distribution (=1.96, approximately)
+	q_975_stdnorm = norm().ppf(0.975)
+
+	std = (q_975 - mean) / q_975_stdnorm # std dev of normal distribution
+
+	# Frozen normal distribution for MD, representing uncertainty in our effect size
+	bw_md_distribution = norm(mean, std)
+
+	# random sample from effect size distribution
+	bw_md_per_10_mg_iron = bw_md_distribution.rvs()
+
+.. note::
+
+	The Haider et al. 2013 paper did not report if the dose of iron was 
+	measured as elemental iron or as an iron compound such as NaFeEDTA. We 
+	operated under the assumption that 10 mg of daily iron consumption, as 
+	referenced in the Haider paper, represented 10 mg of *elemental* iron.
+
+Therefore, we investigated the amount of daily iron consumption that a 
+pregnant mother would likely consume through iron fortification of staple 
+foods in our locations of interest in order to scale the effect size 
+accordingly. See the table below:
+
+.. list-table:: Daily Iron Consumption Parameters
+  :widths: 5 5 5ight
+  :header-rows: 1
+
+  * - Location
+    - Concentration of forticant in flour
+    - Amount of fortifiable flour consumed daily
+  * - Ethiopia
+    - 30 mg NaFeEDTA / kg flour  [1]
+    - 100 g (IQR: 77.5, 200) [2]
+  * - India
+    - 14 to 21.25 mg NaFeEDTA / kg flour [1]; assume midpoint value of **17.6**
```

Comment:
We could model this with a distribution - maybe a symmetric triangular distribution on the interval?

Reply:
Or maybe just a uniform distribution?

Reply:
Sure. Do you think it is worth it?

Reply:
I think we have a couple options here.

The data we have says that there is a range of fortification levels in India, so either:

1. We could use a single number for the mean -- we could either just say this is the midpoint, or we could use e.g. a beta distribution or triangular distribution to model parameter uncertainty, since we're not sure how much fortification happens at each level so we don't really know where the mean is,
2. OR we could we could actually model the range of fortification levels among different households, and use e.g. a uniform distribution to randomly assign a fortification level to each simulant who gets fortified food.

What do you think?

Reply:
I think that we should model it as a population parameter, not an individual parameter since the fortification at a certain concentration is presumably happening at an industrial level.

I was just thinking that because we do not know anything about the underlying distribution that we might want to avoid guessing at it, but if we want to prioritize including the uncertainty in our model, then I would say that option 1 with a uniform distribution would make the most sense to me.

Reply:
Assuming a triangular or beta distribution seems odd to me because for all we know, all of the industries might fortify at the low range of the recommended range (to save money) or the high end (maximize impact). Which is why I would prefer a uniform distribution, I think.

Reply:
Well, if we just choose a single number, that **is** still guessing at the underlying distribution - it's saying that we're 100% sure it's the number we pick. Speaking of which, do we just want to use single numbers for Nigeria and Ethiopia, or should we model uncertainty in those?

I'm not sure I see the argument for using a single parameter rather than assigning different levels to different simulants, other than that it's computationally cheaper. The first option (single parameter) says that everyone gets exactly the same concentration of iron in their fortified food. The second option (different levels to different simulants) says that different factories are fortifying at different levels in the specified range, and each person will get an amount of iron that depends on which factory they get their food from.  To me the second option seems closer to reality.

Reply:
I guess in the second option, we are saying we are 100% sure what the distribution of fortification levels among factories looks like, which is perhaps what you're objecting to.  I guess to avoid that, we'd have to model uncertainty in the *distribution itself*, e.g. by choosing a different beta distribution for each draw (uniform is a special case of beta, so that could be included, but we could also include scenarios where the distribution among factories is asymmetric).

So if we want to avoid that level of complexity, we can say that we really only want to guess at the *mean*, not the entire distribution, and model uncertainty in our estimate of the mean, and then just assign all simulants the mean level of fortification for simplicity.

Reply:
Right, I mean, it's saying in the absence of better data we're making this simple assumption, but yeah.

I don't think we should model uncertainty for Nigeria and Ethiopia because those values are supposed to be national standards (presumably all iron fortified foods should be at that concentration)

And yeah, I agree with what you're saying in the second half there. I guess the key is that the variation will be happening between groups of individuals, not necessarily an individual level, but an individual level is the closest thing we have to that, so I'm fine with doing a uniform distribution at the stimulant level for India.

Also, I just looked back into the source and for non-NaFeEDTA forticants they report an average value for india of 35.25, which is the midpoint of the range. So it's probably reasonable to assume the midpoint is the average for NaFeEDTA forticants too.

Reply:
Haha, I bet they just took the midpoint of 28 and 42.5 to get their "average" of 35.25, rather than estimating it by collecting data. So yeah, I suppose we can do the same thing.

Ok, let's just make it clear what assumptions we're making.